sdks/python/apache_beam/utils/processes_test.py

```diff
@@ -48,11 +48,9 @@ def test_method_forwarding_not_windows(self, *unused_mocks):
     processes.subprocess.check_call.assert_called_once_with(
         ['subprocess', 'check_call'], shell=False, other_arg=True)
 
-    processes.check_output(['subprocess', 'check_output'],
-                           shell=False,
-                           other_arg=True)
```

Comment:
Does this actually trigger the pylint warning? The function accepts arbitrary keyword args: https://github.com/apache/beam/blob/640e6c245a89581578da68b26d6e27b55deb6a80/sdks/python/apache_beam/utils/processes.py#L85

Regardless I think `other_arg` is part of what this test is verifying, so I don't think we should remove it. Let's disable the check instead if necessary.

Reply:
The problem is not with check_output. It that check_output call subprocess.run , in that part is where the pylint check is triggered. 
I'll disable this check as well

Reply:
Oh I see, sorry about that. It looks like [`subprocess.run`](https://docs.python.org/3/library/subprocess.html#subprocess.run) is variadic with `other_popen_kwargs`, but those are likely just passed to [`subprocess.Popen`](https://docs.python.org/3/library/subprocess.html#subprocess.Popen). So there is a restricted set of allowed kwargs.

Now I'm thinking what you did originally is a better resolution.